swarmcg/swarmCG.py

```diff
@@ -20,113 +20,36 @@
 from swarmcg.shared import exceptions
 from swarmcg.simulations.potentials import (gmx_bonds_func_1, gmx_angles_func_1, gmx_angles_func_2,
 	gmx_dihedrals_func_1, gmx_dihedrals_func_2)
+from swarmcg.simulations.vs_functions import (vs2_func_1, vs2_func_2, vs3_func_1, vs3_func_2, vs3_func_3, vs3_func_4, vs4_func_2, vsn_func_1, vsn_func_2, vsn_func_3)
```

Comment:
How about 
"""import swarmcg.simulations.vs_functions as f"""
and then referencing all fucntions as f.[function_name]?

Reply:
right :)